src/core/issues-notes.js

```diff
@@ -118,7 +119,28 @@ function handleIssues($ins, ghIssues, conf) {
           }
         }
         $tit.find("span").text(text);
-        if (report.title) {
+        if (ghIssue && report.title && githubAPI) {
+          const labelsGroup = Array.from(ghIssue.labels)
+            .map(label => {
+              const issuesURL = new URL('issues/', conf.github + '/');
```

Comment:
added this '/' otherwise the urls were like https://github.com/w3c/ instead of https://github.com/w3c/respec